contracts/modules/common/BaseTransfer.sol

```diff
@@ -55,12 +55,17 @@ abstract contract BaseTransfer is BaseFeature {
             
     // *************** Internal Functions ********************* //
     /**
-    * @notice Make sure a contract call is not trying to call a module, the wallet itself, or a supported ERC20.
+    * @notice Make sure a contract call is not trying to call a module, a feature, or the wallet itself.
     * @param _wallet The target wallet.
     * @param _contract The address of the contract.
      */
     modifier onlyAuthorisedContractCall(address _wallet, address _contract) {
-        require(_contract != _wallet && !IWallet(_wallet).authorised(_contract), "BT: Forbidden contract");
+        require(
+            _contract != _wallet && 
+            !IWallet(_wallet).authorised(_contract) && 
```

Comment:
This call is obselete if we assume that the `VersionManager` is the only module since it is also a feature and will be included in `isFeatureAuthorised` . But I don't know if that assumption is worth saving ~3000 gas for a contract call. I'll let you decide.

Reply:
I agree and will remove the check. Though keep in mind that `SimpleUpgrader.init()` can be called then. That's not a problem as the call will fail (since the `SimpleUpgrader` will be unauthorised so the attempt to authorise `toEnable[i]` will revert).